Go/RestApi(No mux)/README.md

```diff
@@ -0,0 +1,9 @@
+### Self parsing RestApi without using mux or any other third party packages
+
+#### To start server run
+
+> $ go run main.go
+
+#### To build the binary
+
+> $ go build
```

Comment:
Try to add the little more about the working of the APAI module to main README.md and also drop some schots here